### PR TITLE
Remove scollbars and change zoom speed on voltage level diagram

### PR DESCRIPTION
**Pull request by FranckLecuyer**
Change zoom factor of panZoom component to increase zoom speed

### CODE REVIEW

File: src/components/single-line-diagram.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ const SingleLineDiagram = (props) => {
                 .addTo(divElt)
                 .size(svgWidth, svgHeight)
                 .viewbox(xOrigin, yOrigin, svgWidth, svgHeight)
-                .panZoom({panning: true, zoomMin: 0.5, zoomMax: 10, zoomFactor: 0.2, margins: {top: svgHeight/4, left: svgWidth/4, bottom: svgHeight/4, right: svgWidth/4}});
```
Comment (jonenst) [inline src/components/single-line-diagram.js:101]: On my mouse, to have a "normal" experience, I need to use zoomFactor = 10 or 20 !  What does it do on your mouse ??

Comment (jonenst) [reply src/components/single-line-diagram.js:101]: even 200 feels natural

Comment (FranckLecuyer) [reply src/components/single-line-diagram.js:101]: With my mouse, when i set the zoom factor to 10, it zooms a lot with only one wheel step ...

Comment (jonenst) [reply src/components/single-line-diagram.js:101]: Maybe we need to add a parameter next to the "center" and "diagonal labels" ?

Comment (FranckLecuyer) [reply src/components/single-line-diagram.js:101]: why not ...

**Pull request merged**